### ISSUE TITLE
Black images

### ISSUE THREAD

**Issue by Rubenacp** [FuouM/ComfyUI-EbSynth#7]
Hi my friend,
Do you have any idea why the images are black? Only the first frame appears, the others are black.
In all workflows the same thing happens.
------
source_video.shape=torch.Size([17, 512, 816, 3])
style_images.shape=torch.Size([2, 512, 816, 3])
Masking mode: False
[0, 16] blend [0, 1]
Calculating edge maps: 100%|██████████████████████████████████████████████████████████| 17/17 [00:00<00:00, 586.39it/s]
F:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-EbSynth\Ezsynth\ezsynth\utils\flow_utils\OpticalFlow.py:54: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  state_dict = torch.load(model_path, map_location=self.DEVICE)
Init Ezsynth took: 0.3526 s
forward Only
Forward mode. start=0, end=16, step=1
Generating:   0%|                                                                               | 0/16 [00:00<?, ?it/s]F:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-EbSynth\Ezsynth\ezsynth\utils\flow_utils\core\ef_raft.py:119: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  with autocast(enabled=self.args.mixed_precision):
F:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-EbSynth\Ezsynth\ezsynth\utils\flow_utils\core\ef_raft.py:132: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  with autocast(enabled=self.args.mixed_precision):
F:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-EbSynth\Ezsynth\ezsynth\utils\flow_utils\core\ef_raft.py:190: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  with autocast(enabled=self.args.mixed_precision):
Generating: 100%|██████████████████████████████████████████████████████████████████████| 16/16 [00:07<00:00,  2.16it/s]
Run took: 7.4093 s
Converting flows: 100%|████████████████████████████████████████████████████████████████| 16/16 [00:01<00:00, 12.84it/s]
Prompt executed in 10.72 seconds

![Captura de pantalla 2024-11-29 023039](https://github.com/user-attachments/assets/7dbdcd3f-8fee-4f8a-b9dd-fe18a929fffc)
-----
 ComfyUI startup time: 2024-11-29 02:11:32.835126
** Platform: Windows
** Python version: 3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:12:12) 

Total VRAM 12288 MB, total RAM 32714 MB
pytorch version: 2.4.1+cu124
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 2060 : cudaMallocAsync
Using pytorch cross attention

**Comment by KeremAydin98** [comment]
I have the same issue

**Comment by stevanisya** [comment]
I have the same issue aswell, 
anybody found a solution ? thx !

**Comment by FuouM** [comment]
Its difficult for me to debug such issues. I also don't have the time to investigate. Really sorry!

**Comment by stevanisya** [comment]
@FuouM aww okay thanks  
if you found anything, please share, i would be happy to have ebsynth on comfyui
have a nice day
